css/style.css

```diff
@@ -0,0 +1,2929 @@
+@font-face {
+  font-family: 'Montserrat' sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  src: local('Montserrat Regular'), 
+  local('Montserrat-Regular'),
+  url('../fonts/montserrat-v14-latin_cyrillic-regular.woff2') format('woff2'), 
+  url('../fonts/montserrat-v14-latin_cyrillic-regular.woff') format('woff'); 
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)